streams/src/test/java/org/apache/kafka/streams/perf/YahooBenchmark.java

```diff
@@ -68,23 +62,26 @@
     static class ProjectedEvent {
         /* attributes need to be public for serializer to work */
         /* main attributes */
-        public String eventType;
-        public String adID;
+        String eventType;
+        String adID;
 
         /* other attributes */
-        public long eventTime;
-        public String userID = UUID.randomUUID().toString(); // not used
-        public String pageID = UUID.randomUUID().toString(); // not used
-        public String addType = "banner78";  // not used
-        public String ipAddress = "1.2.3.4"; // not used
+        long eventTime;
+        /* not used
```

Comment:
Should we just remove these?

Reply:
I don't have strong opinion -- might be nice to keep for documentation (ie, the benchmark describes those fields, but we just don't use them?) Thoughts?

Reply:
That makes sense.  Maybe we could add a couple of words in the comment? 
But it's not a big deal, so it could just stay as is.